src/app/types.ts

```diff
@@ -10,11 +10,13 @@ interface BirdAlert {
     destination: string,
     status: RescueStatus,
     birdStatus: BirdStatus,
-    notes: String,
```

Comment:
oh man, Dart capitalizes String and lowercases boolean I think.. why do humans do this

Reply:
I know I was so confused when I got a type error saying a String cannot be set to string.